--- xpp-1.1/cupshelper.cxx.orig	Wed Jan  9 14:34:17 2002
+++ xpp-1.1/cupshelper.cxx	Fri Oct 31 09:32:55 2003
@@ -67,14 +67,14 @@
 	return port_;
 }
 
-void CupsHelper::setHostInfo(const char *host, int port = 631){
+void CupsHelper::setHostInfo(const char *host, int port /* = 631 */){
 	strcpy(host_,host);
 	cupsSetServer(host_);
 	port_ = port;
 	ippSetPort(port_);
 }
 
-void CupsHelper::setLoginInfo(const char *usr = 0, const char *pwd = 0){
+void CupsHelper::setLoginInfo(const char *usr /* = 0 */, const char *pwd /* = 0 */){
 	strcpy(login_,usr);
 	cupsSetUser(login_);
 	strcpy(password_,pwd);
