--- xpp-1.0/cupshelper.cxx.orig	Sat Jan 27 01:22:17 2001
+++ xpp-1.0/cupshelper.cxx	Thu Oct 30 17:35:36 2003
@@ -177,12 +177,12 @@
 	return port_;
 }
 
-void CupsHelper::setHostInfo(const char *host, int port = 631){
+void CupsHelper::setHostInfo(const char *host, int port /* = 631 */){
 	strcpy(host_,host);
 	port_ = port;
 }
 
-void CupsHelper::setLoginInfo(const char *usr = 0, const char *pwd = 0){
+void CupsHelper::setLoginInfo(const char *usr /* = 0 */, const char *pwd /* = 0 */){
 	strcpy(login_,usr);
 	strcpy(password_,pwd);
 }
